package.json

```diff
@@ -12,7 +12,7 @@
     "co": "sui-mono commit",
     "release": "sui-mono release",
     "release:check": "sui-mono check",
-    "lint": "sui-lint js && sui-lint sass",
```

Comment:
If you do this, the sui-precommit will make changes without adding it to stage...
So the commit will be made with incorrect code.

Reply:
Yep!  I will revert it. And look for a better solución.